frontend/src/components/dashboard/Gallery.js

```diff
@@ -57,6 +58,20 @@ export default function Gallery(props) {
     setFilter(event.target.value);
   };
 
+  let cards;
+  if (artworks) {
```

Comment:
FYI you can have ifs/ternary statements in the return method too. 

`<div>{ artworks ? (....UI component tags if yes...) : (... UI component if no..) }</div>`